tests/cpp_unit.cpp

```diff
@@ -1212,6 +1212,10 @@ DEFINE_TEST(test_cpp_frozen_64) {
 }
 
 DEFINE_TEST(test_cpp_flip) {
+    // We put std::numeric_limits<>::max in parentheses to avoid a
+    // clash with the Windows.h header under Windows.
+    const auto uint32_max = (std::numeric_limits<uint32_t>::max)();
```

Comment:
We can move this constant to the top of the file to reduce line count. The name is unambiguous.